tests/config-tests.js

```diff
@@ -4,39 +4,52 @@ const ajv = new Ajv()
 const fs = require('fs')
 const platforms = require('./../platforms')
 
+function formatErrors (errors) {
+    if (!Array.isArray(errors)) {
+        return ''
+    }
+
+    return errors.map(item => `${item.instancePath}: ${item.message}`).join(', ')
+}
+
 const configs = platforms.map((plat) => {
-    return JSON.parse(fs.readFileSync(`./generated/${plat}-config.json`))
+    return {
+        name: `${plat}-config.json`,
+        body: JSON.parse(fs.readFileSync(`./generated/${plat}-config.json`))
+    }
 })
 
 describe('Config schema tests', () => {
-    it('should have a valid root schema', () => {
-        const schema = JSON.parse(fs.readFileSync('./tests/schemas/root.json'))
-        for (const config of configs) {
-            expect(ajv.validate(schema, config)).to.be.equal(true)
-        }
-    })
-
-    it('should have a vaild feature schema', () => {
-        const schema = JSON.parse(fs.readFileSync('./tests/schemas/feature.json'))
-        for (const config of configs) {
-            for (const featureKey in config.features) {
-                expect(ajv.validate(schema, config.features[featureKey])).to.be.equal(true)
-            }
-        }
-    })
-
-    it('should have valid exception lists', () => {
-        const schema = JSON.parse(fs.readFileSync('./tests/schemas/exception.json'))
-        for (const config of configs) {
-            for (const featureKey in config.feature) {
-                for (const exception of config.features[featureKey].exceptions) {
-                    expect(ajv.validate(schema, exception)).to.be.equal(true)
+    const rootSchema = JSON.parse(fs.readFileSync('./tests/schemas/root.json'))
+    const validateRoot = ajv.compile(rootSchema)
+    const featureSchema = JSON.parse(fs.readFileSync('./tests/schemas/feature.json'))
+    const validateFeature = ajv.compile(featureSchema)
+    const exceptionSchema = JSON.parse(fs.readFileSync('./tests/schemas/exception.json'))
+    const validateException = ajv.compile(exceptionSchema)
+
+    for (const config of configs) {
```

Comment:
this way, we make it easier to find where the bug is and iterate over list of configs only once. The output looks like this: 

![success](https://user-images.githubusercontent.com/985504/141807279-a037a13f-3655-4732-9f84-cc6ebcd63a74.jpg)